### PR TITLE
Forms Automated Test - LPS-133503 | master

### PR DESCRIPTION
**Pull request by ewertonluna**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase [hunk 1]
```diff
@@ -208,14 +208,93 @@ definition {
...
+			fieldName = "Multiple Selection");
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "español-España");
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Selección Múltiple");
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Color ES",
```
Comment (joepak) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase:288]: Hey @ewertonluna the field name should be `fieldName = "Selección Múltiple",`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase [hunk 2]
```diff
@@ -208,14 +208,93 @@ definition {
...
+
+		FormViewBuilder.validateFormMultipleSelectionField(
+			fieldLabel = "Option One",
+			fieldName = "Multiple Selection");
+
+		FormViewBuilder.validateFormMultipleSelectionField(
+			fieldLabel = "Option Two",
+			fieldName = "Multiple Selection");
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
```
Comment (joepak) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase:281]: This step seems redundant. If the language is already english (asserted in line 265), there shouldn't be a need to change it to english before changing it to spanish.

**Review by joepak — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase (hunk 2 above)
Comment (joepak) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase:281]: Nevermind, I tried running the test locally and I see that it fails without this step. Not ideal but acceptable for now. We can look into refactoring this macro at a later date to not need both these steps. Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase (hunk 1 above)
Comment (ewertonluna) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase]: Hi @joepak! Sorry for the mistake. Thank you for pointing it out. Just fixed the problem.

Comment (joepak) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase]: Thank you!

**Comment by joepak** [comment]
Combined and pushed to BChan here: https://github.com/brianchandotcom/liferay-portal/pull/102821

**Pull request closed without merging**